CsvParser/csvParser.go

```diff
@@ -73,10 +75,14 @@ func CsvParser(filePath string) [100]incident.DefaultIncident {
 		avgBadDString := strings.Split(rec[7], " ")
 		avgBadDS, _ := strconv.ParseFloat(avgBadDString[1], 64)
 
+		locationString := strings.Split(rec[2], " ")[1]
+
+		ASN := strings.Split(rec[1], " ")[1]
```

Comment:
Let's change the naming of ASN to be lowecase "asn" so that there is consistency with all the variable names